recipes/codingorf/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "v1.0.0" %}
+{% set sha256 = "09dfe82f8b3d5572d302ab6ce6b87c1ad0f6a281b33163f4870150b126c2ff4e" %}
+
+package:
+  name: codingorf
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - ncbi-genome-download=ncbi_genome_download.__main__:main
+    - ngd=ncbi_genome_download.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+source:
+  url: "https://github.com/Woosub-Kim/codingorf/archive/refs/tags/{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - biopython
+
+test:
+  commands:
+    - codingorf -h
+    - codingorf --help
+    - codingorf ATGATGATG
+    - codingorf ATGTTTTGATTTATGGGGTGA
+    - codingorf AAATGATGATGTTAGGGG
+
+about:
+  home: https://github.com/Woosub-Kim/codingorf
+  summary: "codingorf: The codingorf finds translatable ORFs from an input sequence"
+  license: GPLv3
```

Comment:
can you please use https://spdx.org/licenses/ IDs

Reply:
Thanks for advise!